requirements.txt

```diff
@@ -31,6 +31,7 @@ mock
 mysqlclient==1.3.12
 pdfkit
 Pillow==6.2.0
+psycopg2
```

Comment:
If we are to add psycopg2 to the requirements, which I'm okay with, then its probably best to pin a version.

Alternatively we should have neither psycopg2 or mysqlclient and make the user pick one.

Reply:
removed this from requirements, moved it to .lando.yml since it's specific to that environment